### PR TITLE
[CodeCamp]Add doc of  yolov6_description.md

### PR DESCRIPTION
**Pull request by wzr-skn**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation for this PR and the goal you want to achieve through this PR.

## Modification

Add the Network Structure section doc of yolov6_description.md

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a potential influence on downstream projects, this PR should be tested with downstream projects, like MMDetection or MMClassification.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by hhaAndroid** [comment]
@wzr-skn please sign the CLA

File: docs/zh_cn/algorithm_descriptions/yolov6_description.md [hunk 1]
```diff
@@ -2,7 +2,17 @@
...
+</div>
+
+<div align=center >
+<img alt="YOLOv6-L" src="https://user-images.githubusercontent.com/58845482/208235072-a95b8702-983e-41d7-9915-b10016998d33.png"/>
+图 2：YOLOv6-L 模型结构
+</div>
+
+以上结构图由 wzr-skn@github 绘制。
 
 YOLOv6 有一系列适用于各种工业场景的模型，包括N/T/S/M/L，考虑到模型的大小，其架构有所不同，以获得更好的精度-速度权衡。 此外，还引入了一些 "Bag-of-freebies "方法来进一步提高性能，如自我渐变和更多的训练周期。 在工业部署方面，我们采用QAT与信道蒸馏和图形优化来追求极端的性能（后续支持）。
```
Comment (hhaAndroid) [inline docs/zh_cn/algorithm_descriptions/yolov6_description.md:17]:
```suggestion
YOLOv6 提出了一系列适用于各种工业场景的模型，包括 N/T/S/M/L，考虑到模型的大小，其架构有所不同，以获得更好的精度-速度权衡。 此外，还引入了一些 "Bag-of-freebies " 方法来进一步提高性能，如自我渐变和更多的训练周期。 在工业部署方面，作者还采用 QAT 与信道蒸馏和图形优化来追求极端的性能（后续支持）。
```

Comment (hhaAndroid) [reply docs/zh_cn/algorithm_descriptions/yolov6_description.md:17]: 自我渐变是啥意思？ 信道蒸馏 是啥？ 图形优化 是啥？ 如果中文不好翻译就直接用英文吧

File: docs/zh_cn/algorithm_descriptions/yolov6_description.md [hunk 2]
```diff
@@ -2,7 +2,17 @@
 
 ## 0 简介
 
-以上结构图 xxx 绘制。
+<div align=center >
+<img alt="YOLOv6-S" src="https://user-images.githubusercontent.com/58845482/208234811-e788aa3a-367d-4a6b-89c0-22abc27c69b7.png"/>
```
Comment (RangeKing) [inline docs/zh_cn/algorithm_descriptions/yolov6_description.md:6]: 图的四周留白注意保持等宽

File: docs/zh_cn/algorithm_descriptions/yolov6_description.md [hunk 3]
```diff
@@ -2,7 +2,17 @@
...
+图 1：YOLOv6-S 模型结构
+</div>
+
+<div align=center >
+<img alt="YOLOv6-L" src="https://user-images.githubusercontent.com/58845482/208235072-a95b8702-983e-41d7-9915-b10016998d33.png"/>
```
Comment (RangeKing) [inline docs/zh_cn/algorithm_descriptions/yolov6_description.md:11]: 图的四周留白注意保持等宽

File: docs/zh_cn/algorithm_descriptions/yolov6_description.md [hunk 4]
```diff
@@ -57,12 +67,35 @@ YOLOv6 目标检测算法中使用的数据增强与 YOLOv5 基本一致，唯
 
 ### 1.2 网络结构
 
+YOLOv6 N/T/S 模型的网络结构由 `EfficientRep` + `Rep-PAN` + `Efficient decoupled Head` 构成，M/L 模型的网络结构则由 `CSPBep` + `CSPRepPAFPN` +  `Efficient decoupled Head` 构成。其中，Backbone 和 Neck 部分的结构与 YOLOv5 较为相似，但不同的是其采用了重参数化结构 `RepVGG Block` 替换掉了原本的 `ConvModule`，在此基础上，将 `CSPLayer` 改进为了多个 `RepVGG` 堆叠的 `RepStageBlock`（N/T/S 模型）或 `BepC3StageBlock`（M/L 模型）；Head 部分则参考了 FCOS 和 YOLOX 的检测头，将回归与分类分支解耦成两个分支进行预测。YOLOv6-S 和 YOLOv6-L 整体结构分别如图 1 和图 2 所示。
+
 #### 1.2.1 Backbone
 
+已有研究表明，多分支的网络结构通常比单分支网络性能更加优异，例如 YOLOv5 的 `CSPDarknet`，但是这种结构会导致并行度降低进而增加推理延时；相反，类似于 `VGG` 的单分支网络则具有并行度高、内存占用小的优点，因此推理效率更高。而 `RepVGG` 则同时具备上述两种结构的优点，在训练时可解耦成多分支拓扑结构提升模型精度，实际部署时可等效融合为单个 3×3 卷积提升推理速度，`RepVGG` 示意图如下。因此，YOLOv6 基于 `RepVGG` 重参数化结构设计了高效的骨干网络 `EfficientRep` 和 `CSPBep`，其可以充分利用硬件算力，提升模型表征能力的同时降低推理延时。
+
+<img src="https://user-images.githubusercontent.com/58845482/208235402-782931dc-d517-4f61-8f6d-79a79a78c1d6.png" alt="image" style="zoom:67%;" />
```
Comment (RangeKing) [inline docs/zh_cn/algorithm_descriptions/yolov6_description.md:76]: 图的四周留白注意保持等宽

File: docs/zh_cn/algorithm_descriptions/yolov6_description.md (hunk 1 above)
Comment (PeterH0323) [reply docs/zh_cn/algorithm_descriptions/yolov6_description.md:17]: 改为：

**Comment by PeterH0323** [comment]
Hi @wzr-skn please sign the CLA

**Review by hhaAndroid — APPROVED** [review]

**Comment by OpenMMLab-Team** [comment]
Hi @wzr-skn ！First of all, we want to express our gratitude for your significant PR in the MMYOLO project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/UjgXkPWNqA

If you have WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution！❤

**Pull request merged**